atom/browser/atom_blob_reader.cc

```diff
@@ -28,22 +28,22 @@ void FreeNodeBufferData(char* data, void* hint) {
   delete[] data;
 }
 
-void RunCallbackInUI(const AtomBlobReader::CompletionCallback& callback,
-                     char* blob_data,
-                     int size) {
+void RunPromiseInUI(const atom::util::CopyablePromise& promise,
```

Comment:
Can't we remove this now that we have `.Resolve()` automatically sending things to the right thread?

Reply:
(resolved via slack)